remote_attestation/rust/Cargo.toml

```diff
@@ -5,15 +5,17 @@ authors = ["Ivan Petrov <ivanpetrov@google.com>"]
 edition = "2021"
 license = "Apache-2.0"
 
+[features]
+default = ["alloc"]
+std = ["anyhow/std", "prost/std"]
+alloc = []
```

Comment:
Is this actually needed for anything, or is it more for documentation purposes?

Reply:
It was something I experimented with and forgot to deleted. Deleted now.